### ISSUE TITLE
KVM mission control gesture only works on primary computer

### ISSUE THREAD

**Issue by shareef777** [bitgapp/Cursr#68]
### Describe the Bug/Issue
When pairing two systems with Cursr and using four finger gesture for Mission Control, the gesture is only recognized on the originating device of the trackpad.

### Steps to Reproduce

1. Set trackpad gesture for Mission Control
2. Move cursor to second paired device and use gesture
3. Only primary device Mission Control activates

### Expected Behavior
Expect Mission Control gesture to activate the device that the cursor is active on.

**Screenshots**

### Your Environment
 - OS details: MacOS 14.5
 - Cursr Version: 1.7.1
 - License: Pro (for both devices)

**Comment by ragauskl** [comment]
Yeah.. I know it's a bit annoying but the more complex gesture control (one not mimicking simple mouse events) is simply not implemented yet, if enough people request this I can look into macOS specific support for Mission Control, maybe there are some simple event's I can catch and pass onto another device, but for now this is first report since KVM was added to Cursr about a year ago.

I haven't added this yet as I want to provide touchpad gesture support on all OS and if I remember correctly that at very least this includes writing a custom driver for Linux which will be a lot of work, so for now I had to prioritize tasks that are higher priority and quicker to deal with, ex bugs and features that are required by more people.

So I will get to it - it's definitely in the plan, but first I have to finish some other work. I already have 1.8 fully planned, so maybe in 1.9 or 1.10, sorry for the inconvenience in meantime

**Comment by R-Bose** [comment]
Hi! I'm also interested in the implementation of trackpad gestures across platforms.
In general, there are platform specific trackpad gestures for 2,3 and 4 fingers typically for window interactions. I currently have a mac+mac / mac+win setup, and at-least always miss the 2 finger horizontal swipe for browser back/forward daily.
Just adding my comment here for a +1 vote for this enhancement :)
